### ISSUE TITLE
Explorer crashes when trying to open C:\Windows\Fonts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#79]
```
Which program triggers the problem? Specify the filename if possible.
explorer.exe

What steps will reproduce the problem?
1. Open the explorer
2. Go to your system's fonts-folder
3. Wait a few seconds for the error to appear

What is the expected output? What do you see instead?
I expected to see my installed fonts.

What version of gdipp (including renderer)? On which operating system?
gdipp 0.7.6 on Windows 7 32bit

Please provide any additional information below. Screenshots always help.


```

Original issue reported on code.google.com by `visuald...@googlemail.com` on 23 Jun 2010 at 1:06

**Comment by GoogleCodeExporter** [comment]
```
0.81 seems to fix it
```

Original comment by `yep...@gmail.com` on 24 Jun 2010 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 0.8.0. It is the "Aharoni" font that caused the problem. The 
alef character in the font is abnormal and gdipp didn't check the return value. 
You can either upgrade to the latest version, or put the following code after 
the last gdimm process node, in case you are using Windows XP or whatever:

<font name="Aharoni.*">
  <renderer>0</renderer>
</font>
```

Original comment by `crendk...@gmail.com` on 25 Jun 2010 at 4:17
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hey, just thought I'd throw in that for me, while this is also true (fonts 
folder open -> explorer crashes), it is also true that simply having any folder 
open with a font file in it will cause explorer to crash. As such, adding the 
aharoni lines into the xml doesn't solve this.

This is on Win7 x64, and again, 0.7.6 ... which may mean this is worthless to 
mention. However, given that in my case it is many more than just the specific 
"Aharoni" font that cause the crash - I can't say any font as I simply haven't 
tried that many - it may be useful information.

I'd give the newer build a go, but I'll probably hold on 'till next stable 
release as thankfully I rarely deal with font files, and if I really need to I 
can do it through something else like winrar.


```

Original comment by `Kawi...@gmail.com` on 29 Jun 2010 at 7:25
